src/panels/lovelace/elements/hui-state-icon-element.ts

```diff
@@ -0,0 +1,65 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import "../../../components/entity/state-badge.js";
+
+import { computeTooltip } from "../../../common/string/compute-tooltip";
+import { handleClick } from "../../../common/dom/handle-click";
+import { longPress } from "../common/directives/long-press-directive";
+import { hassLocalizeLitMixin } from "../../../mixins/lit-localize-mixin";
+import { LovelaceElement, LovelaceElementConfig } from "./types.js";
+import { HomeAssistant } from "../../../types.js";
+import { TemplateResult } from "lit-html";
+
+export class HuiStateIconElement extends hassLocalizeLitMixin(LitElement)
+  implements LovelaceElement {
+  public hass?: HomeAssistant;
+  private _config?: LovelaceElementConfig;
+
+  static get properties() {
+    return { hass: {}, _config: {} };
+  }
+
+  public setConfig(config: LovelaceElementConfig): void {
+    if (!config.entity) {
+      throw Error("Invalid Configuration: 'entity' required");
+    }
+
+    this._config = config;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._config || !this.hass!.states[this._config!.entity!]) {

```

Comment:
Shouldnt need the `this._config!` with the `!` here and check for hass here as well